src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/exceptions/ServerErrorException.java

```diff
@@ -10,4 +10,10 @@ public ServerErrorException(
     ) {
         super(message, cause);
     }
+
+    public ServerErrorException(
+        String message
+    ) {
+        super(message);
+    }
```

Comment:
I could not find where this is used but in any case I believe this is a v1 exception we should try to use the v2 ones which have a application+problem json

Reply:
This is not required will remove it.